tripy/nvtripy/config.py

```diff
@@ -38,6 +40,14 @@
 enable_tensorrt_debug = os.environ.get("TRIPY_TRT_DEBUG_ENABLED", "0") == "1"
 tensorrt_debug_path = os.environ.get("TRIPY_TRT_DEBUG_PATH", os.path.join("/", "nvtripy", "tensorrt-dumps"))
 
+tripy_eager_cache: str = export.public_api(
+    document_under="config.rst",
+    autodoc_options=[":no-value:"],
```

Comment:
We should document the default value:
```suggestion
```
The reason it's omitted for `timing_cache_file_path` is that it's OS-dependent